gensrc/thrift/MasterService.thrift

```diff
@@ -64,12 +64,13 @@ struct TTablet {
 
 struct TDisk {
     1: required string root_path
-    2: required Types.TSize disk_total_capacity
-    3: required Types.TSize data_used_capacity
-    4: required bool used
-    5: optional Types.TSize disk_available_capacity
-    6: optional i64 path_hash
-    7: optional Types.TStorageMedium storage_medium
+    2: required Types.TSize data_total_capacity
+    3: required Types.TSize disk_total_capacity
+    4: required Types.TSize data_used_capacity
+    5: required bool used
+    6: optional Types.TSize disk_available_capacity
+    7: optional i64 path_hash
+    8: optional Types.TStorageMedium storage_medium
```

Comment:
For compatibility, you should not change the ordinal of old fields.
It's better to add some comments for these fields. Some fields are not well distinguished from the name

Reply:
1. In this case AvailCapacity refers to the size of the remaining disk, not Doris's left storage space .
2. The second question, I need to test.

Reply:
I think we should change AvailCapacity to the remaining of TotalCapacity, not the Disk left

Reply:
No problem, this is the code of the previous implementation, let me fix it